resources/assets/less/bem/mod.less

```diff
@@ -17,6 +17,18 @@
  */
 
 .mod {
+  .mods & {
```

Comment:
using `.full-size()` seems to work as well (and remove that `flex`/`width`/`height`).